.github/ci-prerequisites.sh

```diff
@@ -0,0 +1,7 @@
+# Reclaim disk space, otherwise we only have 13 GB free at the start of a job
+
+time docker rmi node:10 node:12 mcr.microsoft.com/azure-pipelines/node8-typescript:latest
+# That is 18 GB
+time sudo rm -rf /usr/share/dotnet
+# That is 1.2 GB
+time sudo rm -rf /usr/share/swift
```

Comment:
add `df -h` to see stats about disk usage after the cleanup

Reply:
Before: `/dev/root        84G   68G   16G  82% /`
After: `/dev/root        84G   43G   41G  51% /`